templates/index.html

```diff
@@ -1,31 +1,39 @@
 {% extends "layout.html" %}
 
-{% block page_id %}home{% endblock %}
-{% block title %}Pulse - How federal government domains are meeting best practices on the web.{% endblock %}
-{% block description %}{{ site.description }}{% endblock %}
+{% block page_id %}https{% endblock %}
+{% block title %}HTTPS (M-15-13 and BOD 18-01) - Pulse{% endblock %}
+{% block description %}How federal government domains are doing at deploying HTTPS and HSTS.{% endblock %}
 
 {% block content %}
-<div class="wrapper">
-  <div class="hero">
-  	<h2>{{ site.name }}</h2>
-  	<h3>{{ site.description }}</h3>
-  </div>
 
-  <div id="results">
-    <div class="https">
-    	<div class="https_chart"
-        data-width="287"
-      ></div>
-    	{% include 'includes/https/donut.html' %}
-      <a href="/https/domains/" class="button">View HTTPS Results</a>
-    </div>
-    <div class="dap">
-    	<div class="dap_chart"
-        data-width="287"
-      ></div>
-    	{% include 'includes/analytics/donut.html' %}
-      <a href="/analytics/domains/" class="button">View Analytics Results</a>
-    </div>
+{% include 'includes/https/header.html' %}
+
+<section id="main-content" class="white-bkg">
+
+  <nav class="tab-menu" role="tablist">
+    <a href="/https/domains/" class="tab" role="tab" aria-controls="tab-panel" aria-selected="false">By Domain</a>
```

Comment:
`/https/domains` is no longer a valid route, no?

Reply:
I strive to make the website a terrible, broken experience